tests/golden_outputs/t0_1_2_calls_at_12100_12141

```diff
@@ -1,7 +1,7 @@
 {
     "variant_calls": [
         {
-            "query_interval": [ 12141, 12141 ],
+            "query_interval": [ 12100, 12141 ],
```

Comment:
That's not correct - the query is for single positions 12100 and 12141, not interval \[ 12100, 12141 \]. See run.py comment